src/cpp/server/server_cc.cc

```diff
@@ -23,6 +23,7 @@
 
 #include <grpc/grpc.h>
 #include <grpc/support/alloc.h>
+#include <grpc/support/cpu.h>
```

Comment:
This is not used in the PR?

Reply:
Oops! I went a different route. I'll remove this.